bee-message/src/payload/milestone/option/mod.rs

```diff
@@ -1,8 +1,8 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-mod pow;
-mod receipt;
+pub mod pow;
+pub mod receipt;
```

Comment:
What do you need from these that is not already exposed ?

Reply:
The dto

Reply:
They are accessible through `milestone::dto::{...}`

Reply:
How exactly?
```
error[E0603]: struct `ReceiptMilestoneOption` is private
   --> src/message_handler.rs:6:69
    |
6   |         payload::milestone::{dto::MilestonePayloadDto, option::dto::ReceiptMilestoneOption},
    |                                                                     ^^^^^^^^^^^^^^^^^^^^^^ private struct
    ```

Reply:
How exactly? ```
error[E0603]: struct `ReceiptMilestoneOption` is private
   --> src/message_handler.rs:6:69
    |
6   |         payload::milestone::{dto::MilestonePayloadDto, option::dto::ReceiptMilestoneOption},
    |                                                                     ^^^^^^^^^^^^^^^^^^^^^^ private struct
    ```
  `option::dto::ReceiptMilestoneOption}` also fails

Reply:
But you're trying to import the base types through the dto modules

Reply:
Ah right, my bad